server/src/main/java/org/apache/iotdb/db/engine/cq/ContinuousQueryService.java

```diff
@@ -51,35 +55,80 @@
       ContinuousQueryTaskPoolManager.getInstance();
   private static final long TASK_SUBMIT_CHECK_INTERVAL =
       IoTDBDescriptor.getInstance().getConfig().getContinuousQueryMinimumEveryInterval() / 2;
+
+  private static final String LOG_FILE_DIR =
```

Comment:
It seems that the dir data/system/cq/cqlog.bin is better.

Reply:
Fixed.